vm/qemu/qemu.go

```diff
@@ -154,13 +158,14 @@ var archConfigs = map[string]*archConfig{
 		Qemu:      "qemu-system-ppc64",
 		TargetDir: "/",
 		QemuArgs:  "-enable-kvm -vga none",
+		NetDev:    "virtio-net-pci",
 		CmdLine:   linuxCmdline,
 	},
 	"linux/riscv64": {
 		Qemu:                   "qemu-system-riscv64",
 		TargetDir:              "/",
 		QemuArgs:               "-machine virt",
-		UseNewQemuNetOptions:   true,
+		NetDev:                 "virtio-net-pci",
```

Comment:
@tklauser you used `virtio-net-device` here,  but when I try to use it with my qemu-system-x86_64, it fails with:
```
qemu-system-x86_64: -device virtio-net-device,netdev=net0: No 'virtio-bus' bus found for device 'virtio-net-device'
```
This thread:
https://lists.gnu.org/archive/html/qemu-discuss/2017-02/msg00060.html
suggests that virtio-net-device would require some additional flags and virtio-net-pci is the way to go.
Not sure how virtio-net-device worked for you. Does virtio-net-pci also work for you?

Reply:
`virtio-net-device` worked fine for me with `qemu-system-riscv64`. Didn't test it with `qemu-system-x86_64` though. I just checked with `virtio-net-pci` and it seems to work for `qemu-system-riscv64` as well, so LGTM.